src/domain/reservation/manage/action/ManageReservationsDropdown.js

```diff
@@ -1,27 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { DropdownButton, MenuItem } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 
 import injectT from '../../../../../app/i18n/injectT';
+import ManageReservationsInfoModal from '../information/ManageReservationsInfoModal';
 
 
-const ManageReservationsDropdown = ({ t }) => {
+const ManageReservationsDropdown = ({
+  t, reservation
+}) => {
+  const [isOpen, toggleModal] = useState(false);
   return (
     <div className="app-ManageReservationDropdown">
       <DropdownButton
         id="app-ManageReservationDropdown"
         title={t('ManageReservationsList.actionsHeader')}
       >
-        <MenuItem>{t('ManageReservationsList.actionLabel.information')}</MenuItem>
+        <MenuItem onClick={() => toggleModal(true)}>{t('ManageReservationsList.actionLabel.information')}</MenuItem>
         <MenuItem>{t('ManageReservationsList.actionLabel.approve')}</MenuItem>
         <MenuItem>{t('ManageReservationsList.actionLabel.deny')}</MenuItem>
         <MenuItem>{t('ManageReservationsList.actionLabel.edit')}</MenuItem>
       </DropdownButton>
+
+      {isOpen && <ManageReservationsInfoModal isOpen={isOpen} onHide={toggleModal} reservation={reservation} />}
```

Comment:
The modal should be rendered in the `ManageReservationsPage` and opening it should be handled with callbacks. It's really weird that the modal is handled purely in the dropdown.

Reply:
We want to do the API requests in the `ManageReservationsPage`.

Reply:
The modal need selected `reservation` data when clicked, put it in `ManageReservationPage` will mean we have to have `selectedReservation` also in `ManageReservationsPage` state

Reply:
You can achieve this with callback functions. You don't need the `selectedReservation` in the actions dropdown. You shouldn't have it in there rather.